.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     container:
-      image: quay.io/fedora/fedora:37
+      image: quay.io/fedora/fedora:41
```

Comment:
Do we even need to run "lint" in a container environment? If so, does it make sense to bump this to the dom0's Fedora version and add "matrix strategy"?